ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -821,3 +827,99 @@ def test_clear_job_queue(self, mock_get_drac_client):
                 job_ids=['JID_CLEARALL_FORCE'])
 
             self.assertIsNone(return_value)
+
+@mock.patch.object(drac_mgmt, 'redfish_utils', autospec=True)
+class DracRedfishManagementTestCase(test_utils.BaseDracTest):
+
+    def setUp(self):
+        super(DracRedfishManagementTestCase, self).setUp()
+        self.config(enabled_hardware_types=['idrac'],
+                    enabled_power_interfaces=['idrac-redfish'],
+                    enabled_management_interfaces=['idrac-redfish'],
+                    enabled_inspect_interfaces=['idrac-redfish'])
+        self.node = obj_utils.create_test_node(
+            self.context, driver='idrac',
+            driver_info=DRAC_REDFISH_INFO_DICT)
+
+    def test_get_properties(self, mock_redfish_utils):
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=True) as task:
+            properties = task.driver.get_properties()
+            for prop in redfish_utils.COMMON_PROPERTIES:
+                self.assertIn(prop, properties)
+
+    def test_clear_job_queue(self, mock_redfish_utils):
+        mock_system = mock_redfish_utils.get_system.return_value
+        mock_manager = mock.MagicMock()
+        mock_system.managers = [mock_manager]
+        mock_manager_oem = mock_manager.get_oem_extension.return_value
```

Comment:
Then, I don't think you need any of this - just use mock_get_sushy_oem_manager

Reply:
Addressed.